app/routes/events/components/EventDetail/index.js

```diff
@@ -295,26 +278,50 @@ export default class EventDetail extends Component<Props> {
               </Flex>
             </ContentMain>
             <ContentSidebar>
-              <InfoList items={infoItems} />
+              <div>
+                <Icon name="time-outline" style={{ marginRight: '6px' }} />
+                <strong>
+                  <FromToTime from={event.startTime} to={event.endTime} />
+                </strong>
```

Comment:
Thoughts on having a separate row for date and time? With for example the time-outline icon for time and a calendar icon for date?
I think it is often just as relevant what time of day it is and having it behind day of the week + date + month makes it quite long and hidden-ish. Potentially problematic with events that last multiple days though 🤔

Reply:
Tja. I think it is an "avveining" between reducing number of lines and looking nice. There is quite a lot going on already.
And i think it will be a bit weird with e.g. Kompani Abakus spanning over multiple days and time. It is an edge case tho. What do you think?

Reply:
Fair, probably easiest to keep it as it is as there are quite many cases to handle - and multiday events happen pretty regularly. Possible issue for another PR:)